pkg/client/unversioned/portforward/portforward.go

```diff
@@ -120,7 +120,7 @@ func New(dialer httpstream.Dialer, ports []string, stopChan <-chan struct{}, out
 		dialer:   dialer,
 		ports:    parsedPorts,
 		stopChan: stopChan,
-		Ready:    make(chan struct{}),
+		Ready:    readyChan,
```

Comment:
Could you please add a nil check before we close the channel here: https://github.com/hodovska/kubernetes/blob/9d634168d38d6c853ca3ce1510788f04a19f4a9b/pkg/client/unversioned/portforward/portforward.go#L167?